Cargo.toml

```diff
@@ -24,15 +29,15 @@ serde_qs = "0.13.0"
 axum = { version = "0.7.5", features = ["macros"] }
 alloy = { git = "https://github.com/alloy-rs/alloy", version = "0.3.1", features = [
     "full",
-    "signer-mnemonic"
+    "signer-mnemonic",
 
 ] }
 
 futures-util = "0.3"
 rusqlite-from-row = "0.2.4"
-tower-http = {version = "0.5.2", features =["cors"]}
+tower-http = { version = "0.5.2", features = ["cors"] }
 http = "1.1.0"
-axum-server = {version = "0.6.0", features = ["tls-rustls"]}
+axum-server = { version = "0.6.0", features = ["tls-rustls"] }
```

Comment:
would prefer to review a separate PR for cosmetic changes just to keep the scope of changed files smaller